arcgis-ios-sdk-samples/Scenes/Scene layer selection/README.md

```diff
@@ -1,27 +1,36 @@
 # Scene Layer Selection
```

Comment:
Should this be changed?
```suggestion
# Scene layer selection
```
In plist and the folder it is named correctly, so I think it is safe to change it here?